Tiltfile

```diff
@@ -0,0 +1,181 @@
+        sync('./src/productcatalogservice', '/src'),
+    ],
+)
+
+###
+# Service: currencyservice
+# Desc: Converts one money amount to another currency.
+#       Uses real values fetched from European Central Bank.
+#       It's the highest QPS service
+# Language: Node.js
+###
+docker_build_with_restart('currencyservice',
+    context='./src/currencyservice',
+    entrypoint='node /usr/src/app/server.js',
+    live_update=[
+        sync('./src/currencyservice', '/usr/src/app'),
+        run('cd /usr/src/app && npm install', trigger=['./src/currencyservice/package.json']),
+    ]
+)
+
+###
+# Service: paymentservice
+# Desc: Charges the given credit card info (mock) with the given amount
+#       and returns a transaction ID
+# Language: Node.js
+###
+docker_build_with_restart('paymentservice',
+    context='./src/paymentservice',
+    entrypoint='node /usr/src/app/index.js',
+    live_update=[
+        sync('./src/paymentservice', '/usr/src/app'),
+        run('cd /usr/src/app && npm install', trigger=['./src/paymentservice/package.json']),
+    ]
+)
+
+###
+# Service: shippingservice
+# Desc: Gives shipping cost estimates based on the shopping cart.
+#       Ships items to the given address (mock)
+# Language: Go
+###
+docker_build_with_restart('shippingservice',
+    context='./src/shippingservice',
+    entrypoint='/src/shippingservice',
+    live_update=[
+        sync('./src/shippingservice', '/src'),
+    ],
+)
+
+###
+# Service: emailservice
+# Desc: Sends users an order confirmation email (mock)
+# Language: Python
+###
+docker_build('emailservice',
+    context='./src/emailservice',
+    live_update=[
+        sync('./src/emailservice', '/email_server'),
+        run('cd /email_server && pip install -r requirements.txt',
+            trigger='./src/emailservice/requirements.txt'),
+    ],
+)
+
+###
+# Service: checkoutservice
+# Desc: Retrieves user cart, prepares order and orchestrates the payment,
+#       shipping and the email notification
+# Language: Go
+###
+docker_build_with_restart('checkoutservice',
+    context='./src/checkoutservice',
+    entrypoint='/src/checkoutservice',
+    live_update=[
+        sync('./src/checkoutservice', '/src'),
+    ],
+)
+
+###
+# Service: recommendationservice
+# Desc: Recommends other products based on what's given in the cart
+# Language: Python
+###
+docker_build('recommendationservice',
+    context='./src/recommendationservice',
+    live_update=[
+        sync('./src/recommendationservice', '/recommendationservice'),
+        run('cd /recommendationservice && pip install -r requirements.txt',
+            trigger='./src/recommendationservice/requirements.txt'),
+    ],
+)
+
+###
+# Service: adservice
+# Desc: Provides text ads based on given context words
+# Language: Java
+###
+docker_build_with_restart('adservice',
+    context='./src/adservice',
+    entrypoint='/app/build/install/hipstershop/bin/AdService',
+    live_update=[
+        sync('./src/adservice', '/app'),
+    ],
+)
```

Comment:
**Suggestion:**
Thank you for this pull-request.
But next time, please follow the [Contribution Process outlined in CONTRIBUTING.md](https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/CONTRIBUTING.md#contribution-process).
More specifically:
> For bigger changes:
> Create a [GitHub issue](https://github.com/GoogleCloudPlatform/microservices-demo/issues/new/choose) describing the change before working on the implementation. This is important to avoid potentially having to discard your development efforts.